addOns/authhelper/src/main/java/org/zaproxy/addon/authhelper/AuthUtils.java

```diff
@@ -90,8 +90,29 @@ public class AuthUtils {
     public static final String[] HEADERS = {HttpHeader.AUTHORIZATION};
     public static final String[] JSON_IDS = {"accesstoken", "token"};
     private static final String[] USERNAME_FIELD_INDICATORS = {
-        "email", "signinname", "uname", "user"
+        "email", "signinname", "uname", "user", "name", "nome", "nombre"
     };
+    /* A selection of the most common login label links - just European languages for now - more suggestions appreciated */
+    protected static String[] LOGIN_LABELS_P1 = {
+        "login",
+        "log in",
+        "signin",
+        "sign in",
```

Comment:
you aren't encountering underscore or hyphenated variants?

Reply:
This is the text of link/buttons.

Reply:
I'm still surprised that hyphen wasn't encountered.

Reply:
This is not the name/ID attributes.

Reply:
I know, I still see sign-in or log-in. Infrequent but it comes up.